include/mapbox/geojson_value_impl.hpp

```diff
@@ -0,0 +1,295 @@
+#pragma once
+
+#include <mapbox/geojson.hpp>
+#include <mapbox/geojson/value.hpp>
+
+namespace mapbox {
+namespace geojson {
+
+using error = std::runtime_error;
+
+template <typename T>
+T convert(const value &);
+
+template <>
+point convert<point>(const value &val) {
+    assert(val.is<value::array_type>());
+    const auto &valuePoint = *val.getArray();
+    if (valuePoint.size() < 2) {
```

Comment:
is `valuePoint.size() > 2` a valid value?

Reply:
Spec does not say anything about that. I would prefer to keep it like it is now to be compatible with rapidjson implementation, otherwise, it would be a incompatible change.